client/cypress/e2e/layout.cy.ts

```diff
@@ -23,11 +23,34 @@ Object.values(viewports).forEach(({ device, viewportWidth, viewportHeight }) =>
       cy.get('[data-test=MainLayout__Header]').should('be.visible');
     });
 
-    it('Octant logo redirects to projects view', () => {
+    it('Clicking on Octant logo scrolls view to the top on logo click (proposals view)', () => {
+      cy.scrollTo(0, 500);
+      cy.get('[data-test=MainLayout__Logo]').click();
+      // waiting for scrolling to finish
+      cy.wait(2000);
+      cy.window().then(cyWindow => {
+        expect(cyWindow.scrollY).to.be.eq(0);
+      });
+    });
+
+    it('Clicking on Octant logo redirects to projects view (not proposals view)', () => {
+      cy.get('[data-test=Navbar__Button--Settings]').click();
+      cy.get('[data-test=SettingsView]').should('be.visible');
       cy.get('[data-test=MainLayout__Logo]').click();
       cy.get('[data-test=ProposalsView]').should('be.visible');
     });
 
+    it('Clicking on Octant logo redirects to projects view (not proposals view) with memorized scrollY', () => {
```

Comment:
Please do not mix "projects" and "proposals" naming further. It's already mixed and we will unify it in https://linear.app/golemfoundation/issue/OCT-1395/unify-projects-and-proposals-to-projects-in-the-client.
```suggestion
    it('Clicking on Octant logo redirects to projects view (outside projects view) with memorized scrollY', () => {
```